README.md

```diff
@@ -201,85 +225,34 @@ public class Template
 }
 ```
 
-Otherwise the client will raise a `Notify.Exceptions.NotifyClientException`:
-<table>
-<thead>
-<tr>
-<th>
-
-`error["status_code"]`</th>
-<th>
-
-`error["message"]`</th>
-</tr>
-</thead>
-<tbody>
-<tr>
-<td>
-<pre>429</pre>
-</td>
-<td>
-<pre>
-[{
-    "error": "RateLimitError",
-    "message": "Exceeded rate limit for key type TEAM of 10 requests per 10 seconds"
-}]
-</pre>
-</td>
-</tr>
-<tr>
-<td>
-<pre>429</pre>
-</td>
-<td>
-<pre>
-[{
-    "error": "TooManyRequestsError",
-    "message": "Exceeded send limits (50) for today"
-}]
-</pre>
-</td>
-</tr>
-<tr>
-<td>
-<pre>400</pre>
-</td>
-<td>
-<pre>
-[{
-    "error": "BadRequestError",
-    "message": "Can"t send to this recipient using a team-only API key"
-]}
-</pre>
-</td>
-</tr>
-<tr>
-<td>
-<pre>400</pre>
-</td>
-<td>
-<pre>
-[{
-    "error": "BadRequestError",
-    "message": "Can"t send to this recipient when service is in trial mode
-                - see https://www.notifications.service.gov.uk/trial-mode"
-}]
-</pre>
-</td>
-</tr>
-</tbody>
-</table>
+Otherwise the client will raise a `Notify.Exceptions.NotifyClientException`.
+
+|`error.status_code`|`error.message`|
```

Comment:
Is the term error.message correct? It's an error block? with an error message inside?